friendly-frogs/friendly_frogs/morning/stock_api.py

```diff
@@ -0,0 +1,20 @@
+from yahoo_fin import stock_info as si
+
+class Stock:
+
+    def __getattr__(self, item):
+        try:
+            return si.get_live_price(item)
+        except Exception:
+            return 'No data'
+
+
+def get_stocks_price(you_stocks):
+    for stock in you_stocks:
+        class_object = globals()['Stock']
```

Comment:
Why exactly are you accessing Stock through globals() here? You could just use Stock directly, couldn't you?